### ISSUE TITLE
Choice of headnode instance

### ISSUE THREAD

**Issue by tomdeakin** [clusterinthecloud/support#13]
How cross platform are the headnode instances? It's often very painful if the mgmt architecture doesn't match the compute node architectures. For instance, on AWS the mgmt node is AMD but we might be launching Arm-based compute nodes. This means I have to compile inside a running job, or deal with installing and using cross-compilers.

**Comment by milliams** [comment]
Currently the arch of the head node is fixed to reduce the maintenance burden on us. Our recommendation (and how Chris did it before) is to do the compilation in a running job.

Probably the easiest method is to start an interactive job (for example [the `srun` method here](https://support.nesi.org.nz/hc/en-gb/articles/360001316356-Slurm-Interactive-Sessions)) and do the compilation in there.

Since the head node is usually very small (1 or 2 cores and only a few gigs of RAM) it is not usually suitable for large compilations.

**Comment by tomdeakin** [comment]
Makes sense, thanks. Might make a great summer project for someone to add support for this 😄
